internal/services/containerapps/container_app_environment_resource_test.go

```diff
@@ -25,7 +25,7 @@ func TestAccContainerAppEnvironment_basic(t *testing.T) {
 
 	data.ResourceTest(t, r, []acceptance.TestStep{
 		{
-			Config: r.basic(data),
+			Config: r.basic(data, true),
```

Comment:
Instead up updating the config to take an argument can you remove the property from basic and add it to the complete test instead.